cypress/e2e/calendar/check-monthly-view.cy.js

```diff
@@ -0,0 +1,116 @@
+import { Pane, CalendarCell, Headline, Button, including, Link, not } from '../../../interactors';
+
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+const monthlyCalendarViewExpectedUIValues = calendarFixtures.expectedUIValues.monthlyCalendarView;
+
+describe('Checking the view of calendar on "Monthly calendar view" tab', () => {
+  let testCalendarResponse;
+
+  before(() => {
+    // login and open calendar settings
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point and calendar
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+    });
+  });
+
+  beforeEach(() => {
+    cy.openCalendarSettings();
+    cy.do([
+      Pane('Calendar').find(Link('Monthly calendar view')).click(),
+      Pane('Monthly calendar view').find(Link(testServicePoint.name)).click()
+    ]);
+  });
+
+  after(() => {
+    // delete test calendar
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('tests if the "previous" and "next" buttons of the calendar works as expected', () => {
```

Comment:
`it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {`